tools/GSDumpGUI/Forms/Helper/RichTextBoxLogger.cs

```diff
@@ -0,0 +1,59 @@
+﻿/*
+ * Copyright (C) 2009-2019 PCSX2 Dev Team
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace GSDumpGUI.Forms.Helper
+{
+    public class RichTextBoxLogger : ILogger
+    {
+        private readonly RichTextBox  _richTextBox;
+        public RichTextBoxLogger(RichTextBox  richTextBox)
+        {
+            _richTextBox = richTextBox;
+            _richTextBox.BackColor = Color.White;
+        }
+
+        private void WriteLine(Color color, string line = null)
+        {
+            if (line is null)
```

Comment:
I'm getting a compiler warning here expecting a type.

Reply:
I am using vs2017 most probably you have an older version. Probably this is c#6/7. I can rewrite it without using modern language features.

Reply:
Yes I'm using VS2015. It's a good idea to rewrite it since the project supports 2015.